### PR TITLE
Serverless Challenges

### PR DESCRIPTION
**Pull request by flomotlik**

### CODE REVIEW

File: content/blog/2017-02-27-faas-serverless-enabler.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+Multi part Series on my thoughts on Serverless and its challenges for this year.
+
+Definition of Serverless has been a moving target. Since the launch of Lambda and the start of Serverless as a term we've seen a lot of different ways it has been described. AWS changed their messaging around Serverless last ReInvent going from Serverless == FaaS to Serverless encompasing many different and necessary services they provide. Lambda and FaaS for them is an enabler of Serverless, but not Serverless itself anymore. Check out [Tim Wagners talk on the State of Serverless](https://www.youtube.com/watch?v=AcGv3qUrRC4) for more info there
+
+The main principles of Serverless infrastructure still hold though:
+
+1. Less Control
+2. Less Responsibility
+3. Increased Automation
+4. Gradual Escalation of Control and Responsibility when necessary
```
Comment (johnchapin) [inline content/blog/2017-02-27-faas-serverless-enabler.md]: Another one we like is granular/elastic cost

File: content/blog/2017-02-27-faas-serverless-enabler.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+While we will still be able to test some parts of our system in isolation before roll-out, testing the whole system before every rollout will most likely slow down the rollout process too much. This means we need to start putting a lot more emphasis on post-release validation, monitoring and self-healing of infrastructure. Charity Majors has a great [two part Series on Serverless operations](https://charity.wtf/2016/05/31/operational-best-practices-serverless/) and Mike Roberts talked about this as well in his [on the future of Serverless post](https://www.infoq.com/articles/future-serverless).
+
+AWS provides us all the services we need to make good validation, monitoring and self-healing a reality, but its often hidden behind bad tooling and usability. CloudWatch logs and metrics are powerful systems full of possibilities, but without us as a community, and of course AWS, providing clearer guidance, examples and tooling on how to effective manage Serverless infrastructure with as little manual interaction as possible we won't be able to sufficiently move to Serverless anytime soon.
```
Comment (johnchapin) [inline content/blog/2017-02-27-faas-serverless-enabler.md]: Perhaps mention X-Ray as well?

Comment (flomotlik) [reply content/blog/2017-02-27-faas-serverless-enabler.md]: thanks, included

File: content/blog/2017-02-27-faas-serverless-enabler.md (hunk 1 above)
Comment (flomotlik) [reply content/blog/2017-02-27-faas-serverless-enabler.md]: I changed the sentence before to refer to building serverless infrastructure. Imho costs are an important factor when evaluation and for the success of Serverless infrastructure, but they play a smaller role when building the infrastructure (unless any specific part is prohobitively expensive)

**Pull request merged**